client/src/pages/ProductList/ProductList.tsx

```diff
@@ -0,0 +1,32 @@
+import styled from 'styled-components';
+
+import { ListItem } from 'components/ListItem';
+import useProductList from 'hooks/useProductsList';
+
+const ProductList = () => {
+  const { products, isLoading } = useProductList();
+
+  if (isLoading) {
+    return <div>로딩중...</div>;
+  }
+
+  return (
+    <Layout>
+      {!isLoading &&
+        products &&
+        products.map((product) => (
+          <ListItem key={product.id} price={product.price} image={product.image} name={product.name} />
+        ))}
```

Comment:
합치면 아래처럼 코드를 작성해도 될거 같아요!

```suggestion
      {
        products?.map((product) => (
          <ListItem key={product.id} price={product.price} image={product.image} name={product.name} />
        ))}
```